internal/manifest/plan.go

```diff
@@ -186,9 +187,23 @@ func planDefaultEnv(ctx context.Context, mvars []*EnvVar, client sdkservices.Ser
 		})
 
 		val := mvar.Value
-		if mvar.EnvVar != "" {
+		envVar := mvar.EnvVar
```

Comment:
The var names in this section aren't clear: what's the purpose/difference between `val`, `envVar`, and `mvar`?

Reply:
see the autokitteh.yaml doc